src/main/java/com/zendesk/maxwell/producer/MaxwellKafkaProducer.java

```diff
@@ -111,6 +109,7 @@ String getFallbackTopic() {
 	public MaxwellKafkaProducer(MaxwellContext context, Properties kafkaProperties, String kafkaTopic) {
 		super(context);
 		this.queue = new ArrayBlockingQueue<>(100);
+		BlockingQueue<Runnable> runQueue = new ArrayBlockingQueue<>(100);
```

Comment:
Is `runQueue` used anywhere?

Reply:
Ah nope, I was trying something out and forgot to back this out